development/tools/cmd/synchronizer/syncomponent/component_test.go

```diff
@@ -0,0 +1,162 @@
+package syncomponent
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_RelativePathToComponent(t *testing.T) {
+	// Given
+	path := "/this/is/main/path/this/is/subpath"
+
+	// When
+	result := RelativePathToComponent("/this/is/main/path", path)
+
+	// Then
+	if result != "this/is/subpath" {
+		t.Fatalf("Subpath to component is incorrect: %q", result)
+	}
+}
+
+func Test_daysDelta(t *testing.T) {
+	ts := []struct {
+		unixString string
+		unixTime   int64
+		expected   int
+	}{
+		{unixTimeAsString(3), time.Now().Unix(), 3},
```

Comment:
I bet that nobody will guess that unixTimeAsString subtract number of days from time.Now. Nobody.
That dog neither: 

https://memegenerator.net/img/instances/84188218/unixtimeasstring.jpg

Reply:
hours were passing, function body was changing but name didn't...